resources/commit_messages.md

```diff
@@ -0,0 +1,13 @@
+Commit Messages
+=======
+
+This pratices are based on [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/) article, by Chris Beams
+
+## Commit Messages
+
+  * Separate subject from body with a blank line
```

Comment:
You don't need to indent this list, you can just simply:

```md
* Item 1
* Item 2
```

Reply:
fixed in 83f2a53